src/modules/aws-ebs/sub-modules/config/loadEbsConfig.ts

```diff
@@ -0,0 +1,39 @@
+import { EbsConfig } from "./EbsConfig";
+
+export const isEnvironmentVariableDefined = (environmentVariable: string | undefined): boolean => {
+  return (environmentVariable && environmentVariable !== "undefined") === true;
+};
+
+export const loadEbsConfig = (): EbsConfig => {
+  let sourceAwsRegion: string;
```

Comment:
You might be able to strongly type this based on available AWS regions in the SDK